polkadot-parachains/src/chain_spec.rs

```diff
@@ -196,6 +196,9 @@ fn integritee_genesis_config(
 		balances: parachain_runtime::BalancesConfig {
 			balances: endowed_accounts.iter().cloned().map(|k| (k, 10__000_000_000_000)).collect(),
 		},
+		democracy: DemocracyConfig::default(),
+		council: Default::default(),
+		technical_committee: Default::default(),
```

Comment:
What does default mean here?

Reply:
Does this mean no councillor? Will this be set later via sudo?

Reply:
Yes, exactly. It should be set via sudo or a public referendum. We can ofc also set a default council..?

Reply:
No this is not fine, I just did not understand the concept. Thanks for explaining!

I production, we'd anyhow need a storage migration if we wanted to populate this.

We might want to add 5 dev accounts for development, what do you think?

Reply:
If we really don't need it it development then it is fine. But if we do, then I'd vote to have it set in genesis, as it reduces the wait time by 15 seconds.

Reply:
Ok, lets skip it